spec/job_batches_v2/job_batches_endpoints.yaml

```diff
@@ -387,6 +454,12 @@ x-paths:
             format: uid
             type: string
           description: A unique identifier for a batch.
+        - name: X-SL-ServiceOrigin
```

Comment:
Please remove, as it is only for internal use. Using this header bypasses the throttling. So it is sensitive.